.checkov.yml

```diff
@@ -0,0 +1,8 @@
+directory:
+- .
+download-external-modules: false
+evaluate-variables: true
+external-modules-download-path: .external_modules
+framework: all
+skip-check:
+- CKV2_GHA_1  #Ensure top-level permissions are not set to write-all
```

Comment:
Since most of the jobs are making changes to the files, they need the `write` permissions. That's why some workflows are failing when we change permissions to `read`. There is a ticket in asana to look into this @hugoalmeida264